src/applications/personalization/profile/tests/e2e/address-validation/page-objects/AddressPage.js

```diff
@@ -15,42 +15,62 @@ class AddressPage {
         })
         .check();
     fields.military && cy.get('#root_city').select('FPO');
-    fields.address &&
-      cy
-        .findByLabelText(/^street address \(/i)
-        .clear()
-        .type(fields.address);
-    fields.address2 &&
-      cy
-        .findByLabelText(/^street address line 2/i)
-        .clear()
-        .type(fields.address2);
-    fields.address3 &&
-      cy
-        .findByLabelText(/^street address line 3/i)
-        .clear()
-        .type(fields.address3);
-    fields.city &&
-      cy
-        .findByLabelText(/City/i)
-        .clear()
-        .type(fields.city);
```

Comment:
Chaining multiple functions is now flagged by eslint as a source of flakiness, so the best practice way is to query the element every time a command is issued.

You can make things a little cleaners by aliasing the query as shown in the new changes, but yea, kind of a bummer you have to re-query but 🤷🏼